cmd/operator-sdk/bundle/create.go

```diff
@@ -234,7 +236,14 @@ func (c bundleCreateCmd) runGenerate() error {
 	if err != nil {
 		return fmt.Errorf("error generating bundle image files: %v", err)
 	}
-	if err = copyScorecardConfig(); err != nil {
+
+	// rootDir is the location where metadata directory is present.
+	rootDir := c.outputDir
+	if rootDir == "" {
+		rootDir = filepath.Dir(c.directory)
+	}
+
+	if err = rewriteBundleImageContents(rootDir); err != nil {
```

Comment:
This needs to go into `generate bundle` code as well.

Reply:
Added to `generate bundle`.